api/apps/api/test/project/delete-project-feature.e2e-spec.ts

```diff
@@ -0,0 +1,267 @@
+      v4(),
+    );
+    await fixtures.ThenProjectOrFeatureWasNotFound(
+      result,
+      `Project with id ${nonExistentProjectId} not found`,
+    );
+  });
+
+  test('should return NotFound error when feature does not exist', async () => {
+    const featureId = v4();
+    const projectId = fixtures.projectId;
+    const result = await fixtures.WhenDeletingFeatureForProject(
+      projectId,
+      featureId,
+    );
+    await fixtures.ThenProjectOrFeatureWasNotFound(
+      result,
+      `Feature with id ${featureId}, for project with id ${projectId}, not found`,
+    );
+  });
+
+  test('should not permit updating a given feature when the feature is associated to a different project', async () => {
+    const projectId = fixtures.projectId;
+    const anotherProjectId = fixtures.anotherProjectId;
+    const featureId = await fixtures.GivenBaseFeature(
+      'differentProject',
+      anotherProjectId,
+    );
+    const result = await fixtures.WhenDeletingFeatureForProject(
+      projectId,
+      featureId,
+    );
+    await fixtures.ThenDeleteWasForbidden(
+      result,
+      featureId,
+      `Feature with id ${featureId}, for project with id ${projectId}, cannot be deleted`,
+    );
+  });
+
+  test('should not permit updating a given feature, when it is not a custom feature', async () => {
+    const projectId = fixtures.projectId;
+    const featureId = await fixtures.GivenBaseFeature('customFeature');
+    const result = await fixtures.WhenDeletingFeatureForProject(
+      projectId,
+      featureId,
+    );
+    await fixtures.ThenDeleteWasForbidden(
+      result,
+      featureId,
+      `Feature with id ${featureId}, for project with id ${projectId}, cannot be deleted`,
+    );
+  });
+
+  test('should not permit updating a given feature, when it is still linked to a scenario', async () => {
+    const viewOnlyProjectId = fixtures.anotherProjectId;
+    const featureId = await fixtures.GivenBaseFeature(
+      'linkedFeature',
+      viewOnlyProjectId,
+    );
+    const result = await fixtures.WhenDeletingFeatureForProject(
+      viewOnlyProjectId,
+      featureId,
+    );
+    await fixtures.ThenDeleteWasForbidden(
+      result,
+      featureId,
+      `Feature with id ${featureId}, for project with id ${viewOnlyProjectId}, cannot be deleted`,
+    );
+  });
+
+  test('should not permit updating a given feature, when it is not deletable', async () => {
+    const viewOnlyProjectId = fixtures.anotherProjectId;
+    const featureId = await fixtures.GivenBaseFeature(
+      'notDeletable',
+      viewOnlyProjectId,
+    );
+    const result = await fixtures.WhenDeletingFeatureForProject(
+      viewOnlyProjectId,
+      featureId,
+    );
+    await fixtures.ThenDeleteWasForbidden(
+      result,
+      featureId,
+      `Feature with id ${featureId}, for project with id ${viewOnlyProjectId}, cannot be deleted`,
+    );
+  });
+
+  test('should delete the name of a feature, when permitted', async () => {
```

Comment:
leftover from test copied over from the previous suite, probably?
```suggestion
  test('should delete a feature, when permitted', async () => {
```